tests/routing.py

```diff
@@ -0,0 +1,12 @@
+"""Routing configuration for Django Channels."""
+# from channels.routing import route
+
+from channels.routing import RouteClass
+
+from resolwe.flow.managers import manager
+
+
+manager.update_routing()
+channel_routing = [  # pylint: disable=invalid-name
+    RouteClass(type(manager)),
```

Comment:
Same as in PR #308 in resolwe, use `route_class` here instead of RouteClass. It's a drop-in replacement.